openbb_sdk/sdk/core/openbb_core/app/static/container.py

```diff
@@ -1,9 +1,22 @@
 """Container class."""
+from typing import Union
 from openbb_core.app.command_runner import CommandRunner
+from openbb_core.app.model.obbject import OBBject
+import pandas as pd
 
 
 class Container:
     """Container class for the command runner session."""
 
     def __init__(self, command_runner: CommandRunner) -> None:
         self._command_runner = command_runner
+
+    def run(self, *args, **kwargs) -> Union[OBBject, pd.DataFrame, dict]:
+        """Run a command in the container."""
+        obbject = self._command_runner.run(*args, **kwargs)
+        user_settings = self._command_runner.user_settings
+        python_output = user_settings.preferences.python_output
+        if python_output:
+            return getattr(obbject, python_output)()
+        obbject.extra["user_settings"] = user_settings.dict()
```

Comment:
Isn't it a bit "dangerous" to put the raw credentials there?
Also, need to test to ensure that this doesn't end up on logs!

Reply:
Wouldn't this be a more "cautious" approach to use the `to_chat()` method on the OBBject:

```python

# Never mind if syntax is not exactly correct

openai_key = obb.credentials.openai()
my_data = obb.stocks.load(...)

my_data.to_chat(query="explain me this stuff", key=openai_key)
```

As I type this, I also feel it's kinda meh because it would pretty much be the only functionality that makes you deal with api keys.

Reply:
Yes need to think a bit more about implications of allowing this before merging.